core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -1,11 +1,8 @@
 {% extends 'pages/base.html' %}
 
 {% block maintitle %}
-  {% if title %}
-    {{ title }} - Oppia Editor
-  {% else %}
-    Untitled Exploration - Oppia Editor
-  {% endif %}
+  <div ng-if="title">[title]-Oppia Editor</div>
```

Comment:
Have you done any manual testing on this?
I don't think this will work.

Reply:
How to do that manual testing ? Sir ,please any guidance

Reply:
@jervis446, You can do it via following these steps:
- Start dev server (scripts/start.sh)
- Sign in.
- Create an exploration.
- Go to settings of exploration (in nav bar of exploration editor page)
- Rename the title of the exploration (autosave :) )
- Back to exploration page and check the title of your exploration (move your cursor [here](https://www.wikihow.com/Switch-Tabs-in-Chrome#/Image:Switch-Tabs-in-Chrome-Step-7.jpg))

I hope this may help you.

(_Sorry for late response!_)